robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -76,10 +90,15 @@ public final class DIGITAL_INPUT {
 
   public static final class SWERVE {
 
-    public static final double MAX_ANGULAR_RATE_ROTATIONS_PER_SECOND =
-      Math.PI * 2;
+    public static final AngularVelocity MAX_ANGULAR_VELOCITY =
+      Units.RotationsPerSecond.of(Math.PI * 2);
```

Comment:
@MidgleyM @NolanCampbell2357 If we're wanting this to be the same as last year, I think we want it to be radians per second. I know the constant we were using for this last year says it's in rotations per second but the swerve subsystem and CTRE both use radians per second instead.